modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -90,14 +90,18 @@
 			</return>
 			<argument index="0" name="condition" type="bool">
 			</argument>
+			<argument index="1" name="message" type="String" default="&quot;&quot;">
```

Comment:
Did you add that line manually? It's probably generated automatically, so generating the documentation using `--doctool` again will remove the argument if you added it manually.

Reply:
Thanks! PR updated and tested.